CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+Changelog
+===
+
+### Newer Versions 
+
+See [GitHub releases](https://github.com/jenkinsci/matlab-plugin/releases) for newer versions.
+
+### 1.0.3
+
+Release date: _Nov 05, 2019_
+
+* MATRIX build support. 
+* Plugin support for older versions of MATLAB till R2013a.
+
+### 1.0.2
+
+Release date: _Aug 21, 2019_
+
+* MATRIX build support. 
+* Plugin support for older versions of MATLAB till R2013a.
+
+### 1.0.1
+
+Release date: _May 10, 2019_
+
+* Jenkins run-time environment variables support in MATLAB script.

```

Comment:
This is not very clear. Do you mean
MATLAB script support for Jenkins run-time environment variables.

Reply:
No its other way round. One can define set of environment variables in Jenkins which are visible only in the scope of Jenkins. it will not be applicable for system-level environments. our MATLAB script does support alll such environment variables which are created on the system as well as in Jenkins.